### PR TITLE
PKSD-5086 add wait for workspace runStatus

### PR DESCRIPTION
**Pull request by kgann**
@yijiezhen Can you take a look and also let me know the release flow? 

This is to allow us to wait for terraform cloud workspaces to be applied in an effort to reveal errors should anything go wrong during the run.

### CODE REVIEW

File: wait_for.sh [hunk 1]
```diff
@@ -120,6 +121,16 @@ get_service_state() {
     echo "$get_service_state_states"
 }
 
+# Terraform cloud Workspace runStatus
+# ready when 'applied'
+get_workspace_state() {
+    get_workspace_state_name="$1"
+    get_workspace_state=$(kubectl get workspace "$get_workspace_state_name" -o jsonpath='{.status.runStatus}')
+    if [ "x${get_workspace_state}" != "xapplied" ]; then
+	echo "$get_workspace_state"
```
Comment (yijiezhen) [inline wait_for.sh:130]: we may want to limit the number of the logs (for example echo every 1 of 10 times) if it takes a while for the workspace to come up. Or add a wait after the echo.

Comment (kgann) [reply wait_for.sh:130]: I believe that is already handled via `wait_for_resource` which calls this fn.

**Comment by yijiezhen** [comment]
@kgann we can tag the commit and use quay to manually create the image.

**Review by yijiezhen — APPROVED** [review]

**Comment by yijiezhen** [comment]
It is already set up in quay. Once it is merged, quay will automatically create the image. However, I believe we still need to manually tag it (v1.4 for example) and may need to manually create the image for the tag.

**Pull request merged**